iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/consensus/ConsensusPipeDataNodeDispatcher.java

```diff
@@ -64,6 +68,10 @@ public void createPipe(
       TSStatus status = configNodeClient.createPipe(req);
       if (TSStatusCode.SUCCESS_STATUS.getStatusCode() != status.getCode()) {
         LOGGER.warn("Failed to create consensus pipe-{}, status: {}", pipeName, status);
+        // ignore idempotence logic
+        if (status.getMessage().contains(PIPE_ALREADY_EXIST_MSG)) {
+          return;
+        }
```

Comment:
It seems that this depends on specific error messages. If the error messages on the ConfigNode side are changed, this might stop working. 

Could this using a specific exception or error code instead?

Reply:
fixed, PTAL